sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/ParDoTest.java

```diff
@@ -3938,10 +3939,11 @@ public void testEventTimeTimerOrdering() throws Exception {
       ValidatesRunner.class,
       UsesTimersInParDo.class,
       UsesStatefulParDo.class,
+      UsesUnboundedPCollections.class,
       UsesStrictTimerOrdering.class
     })
     public void testEventTimeTimerOrderingWithCreate() throws Exception {
-      final int numTestElements = 100;
+      final int numTestElements = 5;
```

Comment:
Why shrink it? Does the test get really slow? Is this going to be a perf problem overall?